PocketKit/Sources/PocketKit/Home/SlateDetailViewController.swift

```diff
@@ -50,10 +50,8 @@ class SlateDetailViewController: UIViewController {
         super.init(nibName: nil, bundle: nil)
 
         view.accessibilityIdentifier = "slate-detail"
-
-        title = nil
-        navigationItem.largeTitleDisplayMode = .never
-
+        navigationItem.title = model.slateName
+        navigationItem.setValue(true, forKey: "__largeTitleTwoLineMode")
```

Comment:
Hmm, this seems a bit...unconventional 😆 
Generally speaking using KVC to set a private property is going to be brittle and/or dangerous since the underlying property could go away or get renamed (which would cause a crash here).

What are we trying to accomplish here? Wondering if there's a safer way to get the desired result without having to use a private API

Reply:
updated to using: 

```
        let largeTitleTwoLineMode = "_largeTitleTwoLineMode"
        if class_getProperty(UINavigationItem.self, largeTitleTwoLineMode) != nil {
            navigationItem.setValue(true, forKey: largeTitleTwoLineMode)
        }
```